Pod/Classes/TSMessageView.m

```diff
@@ -146,6 +146,9 @@ + (NSMutableDictionary *)notificationDesign
     if (!_notificationDesign)
     {
         NSString *path = [[NSBundle bundleForClass:self.class] pathForResource:TSDesignFileName ofType:@"json"];
+        if (path == nil) {
+            path = [[NSBundle bundleForClass:[self class]] pathForResource:TSDesignFileName ofType:@"json"];
```

Comment:
That seems to be a boo-boo on my part. I can change that commit to not include that change if you'd like.